sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -84,6 +84,8 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     "date.sql",
     // SPARK-28620
     "postgreSQL/float4.sql",
+    // SPARK-35283
+    "cte-ddl.sql",
```

Comment:
why it doesn't work in thriftserver?

Reply:
Because the output schema of hive is different from spark

Reply:
how about adding a comment here to explain?

Reply:
> how about adding a comment here to explain?

OK

Reply:
Because the output schema of some DDL in Hive is differing from Spark SQL, we exclude it.
For example, the output schema of SHOW TABLES is (namespace, tableName, isTemporary) in Hive, but (tableName) in Spark SQL.